File: bookhub-back/src/main/java/com/example/bookhub_back/dto/book/response/BookResponseDto.java
package com.example.bookhub_back.dto.book.response;

import lombok.*;

import java.time.LocalDateTime;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class BookResponseDto {
    private String isbn;
    private String bookTitle;
    private Long categoryId;
    private String categoryName;
    private String authorName;
    private String publisherName;
    private Long bookPrice;
    private LocalDateTime publishedDate;
    private String coverUrl;
    private String pageCount;
    private String language;
    private String description;
    private String bookStatus;
    private Long policyId;
}

File: bookhub-front/src/dtos/book/response/Book.response.dto.ts
import { BookStatus } from "@/apis/enums/BookStatus";

export interface BookResponseDto {
  isbn: string;
  bookTitle: string;
  categoryId: number;
  categoryName: string;
  authorName: string;
  publisherName: string;
  bookPrice: number;
  publishedDate: Date;
  coverUrl?: string;
  pageCount: string;
  language: string;
  description?: string;
  bookStatus?: BookStatus;
  policyId: number;
}